README.md

```diff
@@ -7,6 +7,8 @@
 
 A framework to generate UI based Splunk Add-ons. It includes UI, Rest handler, Modular input, Oauth, Alert action templates.
 
+> Note: add-ons created by UCC 5 are working starting with Splunk 8.
```

Comment:
I think it after `5.2.0` when it will stop working on Splunk 7.3 because we removed support for python2.

Reply:
I've changed the note message to be more specific.